acceptance/steps/index.test.js

```diff
@@ -22,4 +22,33 @@ JestCucumber.defineFeature(feature, test => {
             driver.quit();
         });
     });
-});
\ No newline at end of file
+
+    test('Index page contains skip to main content link', ({ given, when, and, then }) => {
+        let driver;
```

Comment:
We should make sure we're calling `driver.quit()` on this! It might pass many times but I actually ran into an instance running locally where the login tests timed out and I think this is why.

We should probbaly:
-  move the `driver` variable outside of the test definition, up one level into the feature function
- initialise the `driver` variable in a `beforeEach`
- quit the driver in an `afterEach`

^ this process is already being done in `login.test.js` if you need 'inspiration' 😉

Reply:
@adamgparsons - Can you resolve this one?

Reply:
All done. Thanks Niall for such useful feedback ⭐️